drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -0,0 +1,456 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2016-03-30 16:54+0500\n"
+"PO-Revision-Date: 2016-03-30 16:54+0500\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.7\n"
+"X-Poedit-Basepath: ../../..\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Poedit-SearchPath-0: default_data.py\n"
+"X-Poedit-SearchPath-1: drag_and_drop_v2.py\n"
```

Comment:
@asadiqbal08 Could you remove Poedit-specific info from this block (`X-Generator`, `X-Poedit-Basepath`, `X-Poedit-SearchPath-0`, `X-Poedit-SearchPath-1`)?